src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php

```diff
@@ -50,6 +50,11 @@ class SymfonyTestsListenerTrait
      */
     public function __construct(array $mockedNamespaces = [])
     {
+        if ('0' !== getenv('SYMFONY_PHPUNIT_LOCALE')) {
+            // Enforce a consistent locale
+            setlocale(\LC_ALL, 'C');
+        }
```

Comment:
what about this instead?
```suggestion
        setlocale(\LC_ALL, $_ENV['SYMFONY_PHPUNIT_LOCALE'] ?? 'C');
```

Reply:
That's perfect indeed. I updated the PR